hack/update-bundles.sh

```diff
@@ -120,6 +120,13 @@ for b in $BUNDLES; do
     # Now push
     conftest_push $b "$push_repo:$tag"
 
+    # Add OCI annotations to the bundle image
+    tmpdir="$(mktemp -d --tmpdir)"
```

Comment:
Do we want to trap cleanup this one also?

Reply:
Added a common trap